src/dbwrapper.cpp

```diff
@@ -155,11 +156,28 @@ CDBWrapper::~CDBWrapper()
 
 bool CDBWrapper::WriteBatch(CDBBatch& batch, bool fSync)
 {
+    const bool log_memory = LogAcceptCategory(BCLog::LEVELDB);
+    double mem_before;
+    if (log_memory)
```

Comment:
Nit: braces whenever the then branch is not on the same line.

Reply:
done